src/components/DayPickerRangeController.jsx

```diff
@@ -160,6 +160,8 @@ export default class DayPickerRangeController extends React.Component {
     if (focusedInput !== END_DATE) return false;
 
     if (startDate) {
+      if (minimumNights === 1) return isSameDay(day, startDate);
```

Comment:
![screen shot 2017-02-09 at 7 39 51 pm](https://cloud.githubusercontent.com/assets/5831434/22809368/f0c2e120-eeff-11e6-98a1-69fc044d5948.png)

That did it, also works for other values of minimum nights.